LayoutTests/platform/mac/svg/batik/text/textEffect3-expected.txt

```diff
@@ -99,7 +99,7 @@ layer at (0,0) size 450x500
           RenderSVGInlineText {#text} at (93,2) size 54x69
             chunk 1 (middle anchor) text run 1 at (150.74,340.00) startOffset 0 endOffset 1 width 35.57: "K"
         RenderSVGInlineText {#text} at (0,0) size 0x0
-          [filter="dropShadow"] RenderSVGResourceFilter {filter} at (-45,-50) size 540x600
+          [filter="dropShadow"] RenderSVGResourceFilter {filter} at (0,0) size 0x0
```

Comment:
Where do these changes come from?

Reply:
I tried to see visually using here - https://ews-build.s3-us-west-2.amazonaws.com/macOS-Ventura-Release-WK2-Tests-EWS/ed87b7ae-15732-rerun/retries/svg/batik/text/textEffect3-diffs.html but it seems that it is just text readjustment without any difference in rendering, so I was OK with just rebaseline.